public/app/features/plugins/components/AppRootPage.tsx

```diff
@@ -113,6 +114,7 @@ export function AppRootPage({ pluginId, pluginNavSection }: Props) {
           onNavChanged={onNavChanged}
           query={queryParams}
           path={location.pathname}
+          initialContext={initialContext}
```

Comment:
Maybe a question here is whether to pass it as a prop here or through `PluginContextProvider`. The problem with `PluginContextProvider` though seems that plugins use it to propagate their own plugin context, masking the one that Grafana passes on.